Libertatia 2

Apogee-dev opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
![falmouth_ingame](https://github.com/user-attachments/assets/fe66c0fa-2f79-448f-b3c4-e12d683c9d37)
> "A small, cheap cargo ship most remarkable for its very high speed and maneuverability for a ship of its type. Marketed as a do-it-all light ship suitable for short-range runs and courier work, the Falmouth is extremely popular among smugglers and the occasional ambitious pirate; Falmouths operating on legitimate business are downright rare in some areas, particularly on the Frontier."

Replaces the Libertatia with the Falmouth-class Light Freighter, a ship with a similar design, but a slightly different focus.

![falmouth](https://github.com/user-attachments/assets/da6be93c-d726-4e7d-be29-402d9e881768)
![falmouth_areas](https://github.com/user-attachments/assets/88c3e7b3-ab15-40c7-95fc-8d92ce0db5e5)

The Falmouth shifts the Libertatia's "combat" focus (which had been devastated by power creep anyway) to lean more into the flavor of being a smuggling ship rather than an outright pirate ship. While not a combat ship, the Falmouth _does_ nevertheless come with an eclectic collection of weapons:

- One CM357 Heavy Pistol (stolen)
- One Model 11 Buckmaster Shotgun
- One Candor Pistol
- One E-10 Laser Pistol
- One E-11 Heap Of Garbage

Crew complement consists of a captain, a first mate, a security detail, and two deckhands for a total of 5 crew.

My hope is that the Falmouth can take the broad pattern and flavor of the old Libertatia and make it a much more liveable, balanced, and _playable_ ship.

## Why It's Good For The Game

The Libertatia sucks. This, hopefully, doesn't.

## Changelog

:cl:
add: added Miskilamo Falmouth-class Light Freighter
del: removed Libertatia-class Hauler
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Apogee-dev committed:
draft 1

Apogee-dev committed:
draft 2

Apogee-dev committed:
Update pirate_falmouth.dmm

why are you there

Apogee-dev committed:
Merge remote-tracking branch 'upstream/master' into libertatia-2

Apogee-dev commented:
chat i'm so tired

Apogee-dev committed:
Merge remote-tracking branch 'upstream/master' into libertatia-2

Apogee-dev committed:
manufacturer

Zevotech reviewed (dismissed):
SMES needs the wire under it to be top right instead of right in order to power the ship. The grav gen should also start anchored.
![image](https://github.com/user-attachments/assets/29d639c0-e3dc-4ca5-bf32-a69d7fa12e99)

The exterior thrusters aren't wired to their prechargers.
![image](https://github.com/user-attachments/assets/7a2ab083-b082-4a62-af27-a3ac6d3aaf2b)

This injector needs to be on piping_layer 1 in order to connect to the grey pipe and output gas.
![image](https://github.com/user-attachments/assets/4ab77938-8afc-401e-9e0f-9581a09f10ad)

The safe starts open due to the number of tumblers being 0. Unsure if this is intentional.
![image](https://github.com/user-attachments/assets/009938db-3585-4bf6-8ceb-17b6df14c290)

3 of the SSU's start empty. Use the standard_unit type if you want them to spawn with normal EVA suits.
![image](https://github.com/user-attachments/assets/bc08beb4-1298-4ac1-b111-2b3dee78ddd1)

The Security Detail cannot access the ammunition locker. The Candor in the weapon closet has no magazines anywhere to use for it.
![image](https://github.com/user-attachments/assets/ed6c7c46-bfbc-4435-8a24-31bdc8c12427)


Other than that, Ship looks fine. Can't wait to test it out on live. The doggirls are gonna love this one.

Apogee-dev commented:
>wiring

Fixed.

>pipe layer

It's already set to layer 1.

>safe

gonna try changing to 3 tumblers, I was under the impression number of tumblers was only used to generate the randomized ones

>SSUs

fixed

>ammo locker

Technically intended. the gun locker is ALSO supposed to be locked, but that code seems to be broken. these are both set to use bridge access. that being said i did that when i was still planning to put some armor and other goodies in that locker, so it can change to secoff access

Apogee-dev committed:
fixes

Apogee-dev committed:
locker access

Apogee-dev committed:
removed a duplicate item

Apogee-dev committed:
Merge remote-tracking branch 'upstream/master' into libertatia-2

Erikafox commented:
>tumblers

add an if statement to the tumbler init - if(!tumblers.len()), do the generation of tumblers, and then it'll work with the number set.

Apogee-dev committed:
empty mags

Apogee-dev committed:
visible pipes

Apogee-dev committed:
pickaxe

Apogee-dev committed:
thrusters access

Apogee-dev committed:
ssu access

Apogee-dev committed:
paper

Apogee-dev committed:
extinguishers

Apogee-dev committed:
safe code fix

Apogee-dev committed:
last touches

meemofcourse reviewed (approved):
liberal tail a so good they made a second one
